config.yml

```diff
@@ -14,6 +14,23 @@ relaychain:
     - name: bob
     - name: charlie
 
+statemine:
+  image: parallelfinance/polkadot:v0.9.9-1
+  chain: statemine-local
```

Comment:
do we really have this chain in polkadot image?

Reply:
@GopherJ I am not a docker expert. I considered this https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/src/command.rs#L98

Shall I consider this https://github.com/paritytech/polkadot/blob/v0.9.9-1/cli/src/command.rs#L112?